spec/features/product_category_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'product category dropdown list', js: true do
+  let(:calculator) { create(:calculator) }
+  before do
+    FeatureFlag.get('feature_budget_category').activate
+    visit '/calculator'
+    find(:select, 'product_category')
+    has_select?('product_category', with_options: ['budgetary', 'medium', 'premium'])
+  end
+  
+  it 'default product category' do
+    expect(page).to have_select('product_category', selected: 'medium')
+  end
+
+  it 'custom product category selected' do
+    select('budgetary', from: 'product_category')
+    expect(page).to have_select('product_category', selected: 'budgetary')
```

Comment:
extend this test to check that it does correct calculation results based on different diapers price categories

Reply:
Is no calculation results in my branch with this feature. Is another task [ZER-154](https://github.com/ita-social-projects/ZeroWaste/pull/242)